src/opencl_common.c

```diff
@@ -2943,7 +2950,7 @@ void opencl_list_devices(void)
 			p = dname;
 			while (isspace(ARCH_INDEX(*p))) /* Intel quirk */
 				p++;
```

Comment:
Unrelated to the changes, but: the "Intel quirk" comment is wrong (it's not allowed to use ctype macros on "char" - they're to be used on "int" - and Intel isn't to blame), and we should have this code depend on current locale. So this code is ideally changed to:

```c
while (*p == ' ' || *p == '\t')
        p++;
```